docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -297,11 +297,11 @@ Then elements located `By.xpath(//span[@id='item-to-purchase'])` are sorted by t
 ----
 
 
-=== Take a screenshot
+=== Take a screenshot in Allure Report
```

Comment:
```suggestion
=== Take a screenshot
```
we may use different reporting engines, so UI steps should not declare any explicit reporting engine